src/pages/VestingClaim.tsx

```diff
@@ -118,83 +118,107 @@ function VestingClaim(): ReactElement {
   }
 
   return (
-    <>
-      <div>
-        <div></div>
-        <div>
-          <div>
-            <img src={logo} />
-          </div>
-        </div>
-        <div>
-          {isValidBeneficiary ? (
-            <>
-              <div>
+    <div>
```

Comment:
not to be too nitpicky but these outside divs seem to be unnecessary too

Reply:
removed!